torchvision/io/_video_opt.py

```diff
@@ -113,8 +117,11 @@ def _fill_info(vtimebase, vfps, vduration, atimebase, asample_rate, aduration):
     return meta
 
 
-def _align_audio_frames(aframes, aframe_pts, audio_pts_range):
-    # type: (torch.Tensor, torch.Tensor, List[int]) -> torch.Tensor
+def _align_audio_frames(
+    aframes: torch.Tensor,
+    aframe_pts: torch.Tensor,
```

Comment:
Are we sure that's a Tensor and not a tuple of ints?

Reply:
Well tracing it back, it looks like so https://github.com/pytorch/vision/blob/7947fc8fb38b1d3a2aca03f22a2e6a3caa63f2a0/torchvision/csrc/io/video_reader/video_reader.h#L29

`aframe_pts` is one of the outputs of this function